### ISSUE TITLE
[A11Y] Slides carousel: Unclear what 'Next', 'Previous' refer to

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#5486]
Their text should be changed to `Next item in carousel` and `Previous item in carousel`

Applies to both type="carousel" and type="slides"

**Comment by aghassemi** [comment]
/to @camelburrito an extra one for you
